tests/jballerina-unit-test/src/test/resources/test-src/functions/functions_with_default_parameters.bal

```diff
@@ -190,6 +191,22 @@ function foo7(string s, float b = 1.1, FooObject o = new("default", 100)) return
     return [s, b, o];
 }
 
+function testFuncWithDefaultByteValue() {
+    byte b = funcWithDefaultByteValue();
+    test:assertEquals(b, 10);
+}
+
+function testDefaultByteValueInFunctionPointers() {
+    function (byte) returns byte fp = funcWithDefaultByteValue;
+    byte b = fp(15);
```

Comment:
Shall we add a test for call testFuncWithDefaultByteValue without value?

Reply:
Addressed with https://github.com/ballerina-platform/ballerina-lang/pull/31612/commits/9ae22f2369e7867d43fd390bd0fb21a74e7f35a6